mri/test/test_sensitivity_extraction.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+import unittest
+import numpy as np
+
+# Package import
+from mri.reconstruct.fourier import NFFT
+from mri.reconstruct.utils import convert_mask_to_locations
+from mri.parallel_mri.extract_sensitivity_maps \
+    import get_Smaps, extract_k_space_center_and_locations
+
+
+class TestSensitivityExtraction(unittest.TestCase):
+    """ Test the code for sensitivity extraction
+    """
+
+    def setUp(self):
+        """ Initialization of variables for tests:
+            N = Image size in 2D
+            Nz = Number of slices
+            num_channel = Number of channels
+            percent = percent of kspace to extract while testing
+        """
+        self.N = 64
+        self.Nz = 60
+        self.num_channel = 3
+        # Percent of k-space center
+        self.percent = 0.5
+
+    def test_extract_k_space_center_3D(self):
+        """ Ensure that the extracted k-space center is right"""
```

Comment:
Could you be more explicit in the documentation?

Reply:
Done